src/devices/tuya.ts

```diff
@@ -535,13 +535,24 @@ const definitions: Definition[] = [
         exposes: [e.gas(), e.tamper()],
     },
     {
+        fingerprint: [{modelID: 'TS0205', manufacturerName: '_TZ3210_up3pngle'}],
         zigbeeModel: ['TS0205'],
         model: 'TS0205',
         vendor: 'TuYa',
         description: 'Smoke sensor',
-        whiteLabel: [{vendor: 'Tesla Smart', model: 'TSL-SEN-SMOKE'}],
+        whiteLabel: [
+            {vendor: 'Tesla Smart', model: 'TSL-SEN-SMOKE'},
+            {vendor: 'Dongguan Daying Electornics Technology', model: 'YG400A (Zigbee)'},
+        ],
         fromZigbee: [fz.ias_smoke_alarm_1, fz.battery, fz.ignore_basic_report],
         toZigbee: [],
+        configure: async (device, coordinatorEndpoint, logger) => {
+            try {
+                const endpoint = device.getEndpoint(1);
+                await reporting.bind(endpoint, coordinatorEndpoint, ['genPowerCfg']);
+                await reporting.batteryPercentageRemaining(endpoint);
+            } catch (error) {/* Fails for some*/}
```

Comment:
How did you know it fails for some?

Reply:
Actually, I don't know, copied it from a similar device.

Reply:
Then let's leave it out.

Reply:
@pixeldoc2000 I've already submitted a [PR](https://github.com/Koenkk/zigbee2mqtt.io/pull/2406) picture for this device named "YG400A", could you remove the "(Zigbee)" from the model name ?